bridge/e2e/eth.go

```diff
@@ -105,3 +106,78 @@ func testEthereumLockup(t *testing.T, ctx context.Context, ec *ethclient.Client,
 	// Source account decreases by the full amount.
 	waitEthBalance(t, ctx, token, beforeErc20, -int64(amount))
 }
+
+func testEthereumToTerraLockup(t *testing.T, ctx context.Context, ec *ethclient.Client, kt *bind.TransactOpts,
+	tc *TerraClient, tokenAddr common.Address, isNative bool, amount int64, precisionLoss int) {
+
+	// Bridge client
+	ethBridge, err := abi.NewAbi(devnet.GanacheBridgeContractAddress, ec)
+	if err != nil {
+		panic(err)
+	}
+
+	// Source token client
+	token, err := erc20.NewErc20(tokenAddr, ec)
+	if err != nil {
+		panic(err)
+	}
+
+	// Store balance of source ERC20 token
+	beforeErc20, err := token.BalanceOf(nil, devnet.GanacheClientDefaultAccountAddress)
+	if err != nil {
+		t.Log(err) // account may not yet exist, defaults to 0
```

Comment:
Needs to be either t.Fatal or set `beforeErc20 = new(big.Int)` (this is wrong in the original function too - feel free to fix all occurrences)